domains/analogsombra.json

```
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "analog-sombra",
+        "email": "analogsombra@gmail.com"
+      },
+      "record": {
+        "CNAME": "analogsombra.dev"
```

I can't reach this address though. If you want to use GitHub Pages please change this to `analog-sombra.github.io`.
```suggestion
        "CNAME": "analog-sombra.github.io"
```